### PR TITLE
chore(illustratedmessage)!: update colors, weights, sizes

### LINKED ISSUE #1399: Express Illustrated Message
Same for Spectrum and Spectrum for CCX

- Update font size from 700 light to 500 default
- Update body text style from italic to normal
- Update text from title case to sentence case
- Update heading text style from gray 700 to 800

Just Spectrum

- Heading: Bold (Default) weight for reg spectrum

Just Spectrum for CCX

- Heading: the Express theme will turn Bold text to Black (a thicker weight)

 

Design Paper documentation: https://paper.dropbox.com/doc/Illustrated-message-betaupdate--BdNWZTEvUZYdjMCD6OfMsXPqAg-SeYUHX0wDyedexngS8zwv

 

Dropbox XD File: https://www.dropbox.com/home/Spectrum%20Team/Contributions/Components/Illustrated%20Message?preview=illustrated-message.xd

 

synced to jira: https://jira.corp.adobe.com/browse/CSS-14

### PR DESCRIPTION
**Pull request by pfulton**
BREAKING CHANGE: changes heading font size, body font-style, heading color, and heading font-weight. Also updates docs example to prefer sentence case over title case.

Closes #1399

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

### For both Spectrum & CCX:
- [x] Update font size from 700 light to 500 default
- [x] Update body text style from italic to normal (no change needed—was already `normal`)
- [x] Update text from title case to sentence case (not relevant to DSS. CSS-only change)
- [x] Update heading text style from gray 700 to 800

### For Spectrum only:
- [x] Heading: Bold (Default) weight for reg spectrum

### For CCX only:
- [x] Heading: the Express theme will turn Bold text to Black (a thicker weight)

### Misc.:

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

### CODE REVIEW

**Comment by lazd** [comment]
LGTM! Also agreed that this is a breaking change due to the markup change (i.e. existing markup has to be manually updated for things to be correct).

File: components/illustratedmessage/metadata/illustratedmessage.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ examples:
...
-        <h2 class="spectrum-Heading spectrum-Heading--sizeL spectrum-Heading--light spectrum-IllustratedMessage-heading">Error 404: Page Not Found</h2>
+        <h2 class="spectrum-Heading spectrum-Heading--sizeL spectrum-Heading--light spectrum-IllustratedMessage-heading">Error 404: Page not found</h2>
```
Comment (lazd) [inline components/illustratedmessage/metadata/illustratedmessage.yml:10]: Will this need to change to `--sizeM`?

Comment (pfulton) [reply components/illustratedmessage/metadata/illustratedmessage.yml:10]: Yes. Thank you for catching this!

File: components/illustratedmessage/index.css [hunk 2]
```diff
@@ -34,19 +34,22 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-IllustratedMessage-heading {
+  color: var(--spectrum-illustratedmessage-heading-color);
+  font-size: var(--spectrum-illustratedmessage-heading-font-size);
+  font-weight: var(--spectrum-illustratedmessage-heading-font-weight);
   max-inline-size: var(--spectrum-illustrated-message-heading-max-width);
   margin: var(--spectrum-illustrated-message-heading-margin);
```
Comment (lazd) [inline components/illustratedmessage/index.css:41]: Interesting that some of these are `--spectrum-illustrated-message` and some are `--spectrum-illustratedmessage` -- are we sure no tokens are missing?

Comment (pfulton) [reply components/illustratedmessage/index.css:41]: This is a good call-out. The `spectrum-illustrated-message` ones are defined locally, at the top of that `index.css` file, while the ones from DSS come through as `spectrum-illustratedmessage`. Perhaps we should update the locally defined ones to match the DSS output.

Thoughts?

Comment (lazd) [reply components/illustratedmessage/index.css:41]: It's fine either way honestly, I just wanted to make sure things are building without error. Maybe go ahead and rename them for consistency, then let's get this merged!

**Review by lazd — CHANGES_REQUESTED** [review]
Rename vars to have the same `--spectrum-illustratedmessage-*` prefix and this is good!

**Comment by pfulton** [comment]
> Rename vars to have the same `--spectrum-illustratedmessage-*` prefix and this is good!

Done! Thanks :-)

**Review by lazd — APPROVED** [review]

**Pull request merged**